docs/api-usage.md

```diff
@@ -72,13 +73,11 @@ end
 - This "show" endpoint is the natural mapping for something like `GET /manage/questionnaires/1.json`
 - Because `respond_with` is used, it will return JSON when `.json` is used in the URL
 
-# Authentication
-
-Authentication is implemented with OAauth 2, provided by the [Doorkeeper](https://github.com/doorkeeper-gem/doorkeeper) gem.
+## Authentication
 
-An OAuth app should be created by an admin at https://apply.your-hackathon.com/oauth/applications. From there, you can implement a standard OAuth 2 flow.
+Authentication is implemented with OAauth 2, provided by the [Doorkeeper](https://github.com/doorkeeper-gem/doorkeeper) gem. For full Doorkeeper endpoints + docs, see the [Doorkeeper wiki](https://github.com/doorkeeper-gem/doorkeeper/wiki/API-endpoint-descriptions-and-examples).
 
-For Doorkeeper endpoints + docs, see https://github.com/doorkeeper-gem/doorkeeper/wiki/API-endpoint-descriptions-and-examples
+For a guide to set up & test the API using Postman or Paw, see the [API Setup](api-setup.md) page.
```

Comment:
"API Setup" should probably be "Setup Postman/Paw for API Testing" to match page name